scalafix-core/src/main/scala/scalafix/rule/RuleName.scala

```diff
@@ -25,7 +25,7 @@ final case class RuleName(identifiers: List[RuleIdentifier]) {
   def value: String =
     if (isDeprecated) identifiers.mkString("+")
     else if (nonDeprecated.isEmpty) "empty"
-    else nonDeprecated.mkString("+")
+    else nonDeprecated.head.value
```

Comment:
If you feel like

```scala
if (isDeprecated) identifiers.mkString("+")
else nonDeprecated.headOption.map(_.value).getOrElse("empty")
```

Reply:
Refactored into pattern match to avoid `head` ;)